### PR TITLE
Closes #1004: Integrating ThumbnailsFeature.

### LINKED ISSUE #1004: FNX-4712 ⁃ Integrate ThumbnailsFeature from AC
This will allow users to see thumbnails of the website that they visit.

### PR DESCRIPTION
**Pull request by Amejia481**

### CODE REVIEW

**Comment by boek** [comment]
@Amejia481 Looks like bitrise has an old version of A-C. Purging the cache and rebuilding for you

**Comment by Amejia481** [comment]
Thanks @boek !

File: app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt [hunk 1]
```diff
@@ -256,6 +265,10 @@ class BrowserFragment : Fragment(), BackHandler {
         promptsFeature.withFeature { it.onActivityResult(requestCode, resultCode, data) }
     }
 
+    override fun onTrimMemory(level: Int) {
+        requireComponents.core.sessionManager.onLowMemory()
+    }
```
Comment (pocmo) [inline app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt]: This looks like something that should happen at the "app level" and not be buried somewhere in a Fragment. [Application](https://developer.android.com/reference/android/app/Application) already implements `ComponentCallbacks2` - so you can just override `onTrimMemory` there. :)

Comment (Amejia481) [reply app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt]: 👍

Comment (Amejia481) [reply app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt]: https://github.com/mozilla-mobile/fenix/pull/1018

**Pull request merged**